CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## [Unreleased]
 
+### Added
+- Layer integration tests for more apps. Closes
+  [[#472](https://github.com/metalbear-co/mirrord/issues/472)].
```

Comment:
This PR adds a test with go 1.19.0. Do we also need a test with go 1.18.0? In that case I need to change the changelog entry:
```suggestion
- Layer integration tests for more apps. Addresses but does not close
  [[#472](https://github.com/metalbear-co/mirrord/issues/472)].
```